### DISCUSSION TITLE
Ipad web page map only small horizontal strip

### DISCUSSION THREAD

**Discussion by albar965** [albar965/littlenavmap#787] [Q&A]
@u-an-i Maybe you have an idea what's going there in Safari:

https://www.avsim.com/forums/topic/608191-ipad-web-page-map-only-small-horizontal-strip/

**Comment by u-an-i** [comment]
Below is how it can look on iPad -- depending on theme settings and iPad OS version.

![IMG_0029](https://user-images.githubusercontent.com/84718885/135887040-a5d21676-8f3c-4eee-8b02-09cca3e34043.PNG)


Web UI working on Server indicates all files are present.
The described look lets appear the Stylesheet or/and the JavaScript is not delivered.
I assume this can be due to the network management hardware (router, switch, ...).
To verify the cause is those file(s) not being delivered, append "/assets/styles-2021-a-layout-internal.css" to the url on iPad:

![IMG_0030](https://user-images.githubusercontent.com/84718885/135887021-ae92fd96-613d-4b0a-959b-4e4d8fbe764f.PNG)


and "/assets/scripts-2021-a.js" to the url on iPad:

![IMG_0031](https://user-images.githubusercontent.com/84718885/135887132-8b185758-ef14-4bb3-99fb-e4f7162c37e9.PNG)


If the files content doesn't show up some network hardware/software needs configuration -- or possibly a Win 10 software firewall or "similar".

The domain shown in my screenshots is my router url, the number is my port configured in LNM and enabled as passthrough map in my router.

I have no idea how i can help configuring your network if this is the issue.

If this isn't the issue i currently have no idea what else could be broken; perhaps you find some more clues you can write me as hints?

**Comment by ghost** [comment]
Humble apologies for not replying sooner.... and thank you for your response to my question. Well, it is definitely an iPad issue. Your software works perfectly on another desktop (linux Mint) distro in web mode. Using either Safari or Firefox on the ipad has the same scrunched up result.
Adding the suffix you supplied did return a response showing all the screen settings as percentages for screen size, all either zero or 100 percent. As stated earlier I am new to iPad and will need to get to grips with screen setup etc.
Thank you for your time and trouble.
Best regards
Gordon

**Comment by u-an-i** [comment]
Hello, I tested on an iPad Mini 3 running iOS 8 and Safari and the web ui looked fine wrt the map.
I also disabled JS and CSS in my desktop browser and didn't reproduce the look you described.

iOS 8 Safari did have an issue wrt the toolbar thus I don't rule out even earlier iOS versions have issues wrt the whole web ui.

Which i(Pad)OS version are you running. If below 8 try upgrading. Current is 15.

iPad Settings app offers some settings for Safari. Try clearing cache(s) and resetting as many settings as offered to be resetted if you do not know for sure you like to have a certain setting. Perhaps an experimental Safari setting (which are available) got enabled or there is some cache issue.

Safari and Firefox showing the same issue would indicate it's not the browser. Your other web client working indicates it's not the network (although network software might allow filters for particular devices). Thus it appears to be "an iPad issue". But it's likely not JS or CSS filtered (by a software firewall perhaps) because those not received does not create the look you described.

Try rotating the device? Does the web ui show the same issue in portrait and landscape? Can you attach a screenshot?

**Reply by u-an-i** [reply]
@mbvet see previous reply.
